### ISSUE TITLE
Missing dependencies for opencv-python ?

### ISSUE THREAD

**Issue by windy54** [bennuttall/piwheels#94]
I am using a PI3 with a clean image of stretch and have installed all updates.

I installed opencv using 
    sudo pip3 install opencv-python 
and
    sudo pip3 install opencv-contrib-python
I checked the installation by opening the python 3 IDE and entering import cv2.

This resulted in errors of the form "Import error: libhdf5-serial.so.100" and similar for other files.
I fixed it by entering the following commands:
    sudo apt-get install libatlas-base-dev -y
    sudo apt-get install libhdf5-dev -y
    sudo apt-get install libhdf5-serial-dev -y 
    sudo apt-get install libjasper-dev -y
    sudo apt-get install libqtgui4 -y
    sudo apt-get install libqt4-test -y

So I am not sure whether you would class this as an error with piwheels as such or a problem with the instructions on how to install opencv using piwheels.

(I found the instructions on the PI-wars website and I am going to mention it to them as well.) 

Whatever you decide to do, a great job with PI-wheels, even with having to solve the issues with the missing files, it was a lot quicker than following the pyimagesearch.com instructions I normally use and build from source.

Steve Gale

**Comment by bennuttall** [comment]
@waveform80 do you know what's missing?

**Comment by windy54** [comment]
No, I only made of note of what I had to install to fix it. I do not mind doing it again and give you a listing of the errors if that helps you.
